app/models/workers_scaling/algorithm_runner.rb

```diff
@@ -0,0 +1,77 @@
+##
+# Scheduler to run WorkersScalingAlgorithm decision loop periodically
+module WorkersScaling
+  class AlgorithmRunner
+
+    ##
+    # Returns instance of AlgorithmRunner for experiment_id if registered
+    # Otherwise returns nil
+    def self.get(experiment_id)
+      return @@cache[experiment_id] if @@cache
```

Comment:
Jedna małpa też zadziała, jednak będzie znaczyła coś innego niż dwie. http://www.railstips.org/blog/archives/2006/11/18/class-and-instance-variables-in-ruby/
W tym wypadku dałbym 2.